File: MusicLibraryApp/src/main/java/com/MusicApp/model/Playlist.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.MusicApp.model;

/**
 *
 * @author anshul
 */
public class Playlist {
    
    private int playistId;
    
    private int musicId;
    
    private int userId;
    
    private boolean archived;
    
    public int getPlayistId() {
        return playistId;
    }

    public void setPlayistId(int playistId) {
        this.playistId = playistId;
    }

    public int getMusicId() {
        return musicId;
    }

    public void setMusicId(int musicId) {
        this.musicId = musicId;
    }

    public int getUserId() {
        return userId;
    }

    public void setUserId(int userId) {
        this.userId = userId;
    }

    public boolean isArchived() {
        return archived;
    }

    public void setArchived(boolean archived) {
        this.archived = archived;
    }
}

File: MusicLibraryApp/src/main/webapp/js/playlist.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

var audio;
var playlist;
var tracks;
var current;

init();
$("#add,#all,#searchMusic").click(function () {
    audio = $('#audio');
    playlist = $('#playlist');
    tracks = playlist.find('li a');
    audio[0].pause();
});
function init(){
    current = 0;
    audio = $('#audio');
    playlist = $('#playlist');
    tracks = playlist.find('li a');
    len = tracks.length - 1;
    audio[0].volume = .50;
    audio[0].pause();
    playlist.find('li a').click(function(e){
        e.preventDefault();
        link = $(this);
        current = link.parent().index();
        run(link, audio[0]);
    });
    audio[0].addEventListener('ended',function(e){
        current++;
        if(current == len){
            current = 0;
            link = playlist.find('li a')[0];
        }else{
            link = playlist.find('li a')[current];    
        }
        run($(link),audio[0]);
    });
}
function run(link, player){
        player.src = link.attr('href');
        par = link.parent();
        par.addClass('active').siblings().removeClass('active');
        audio[0].load();
        audio[0].play();
}